cmd/catalyst-node/catalyst-node.go

```diff
@@ -334,9 +314,34 @@ func main() {
 		}
 	}()
 
+	// Everything past this is booting up Serf
+	tmpFile, err := writeSerfConfig(&serfConfig)
+	if err != nil {
+		glog.Fatalf("Error writing serf config: %s", err)
+	}
+	defer os.Remove(tmpFile)
+	glog.V(6).Infof("Wrote serf config to %s", tmpFile)
+
+	ui := &cli.BasicUi{Writer: os.Stdout}
+
+	// eli note: this is copied from here:
+	// https://github.com/hashicorp/serf/blob/a2bba5676d6e37953715ea10e583843793a0c507/cmd/serf/commands.go#L20-L25
+	// but we should someday get a little bit smarter and invoke serf directly
+	// instead of wrapping their CLI helper
+
```

Comment:
```suggestion
	// copied from:
	// https://github.com/hashicorp/serf/blob/a2bba5676d6e37953715ea10e583843793a0c507/cmd/serf/commands.go#L20-L25
	// we should consider invoking serf directly instead of wrapping their CLI helper
```
Nit: I suggest to not `eli note` in the comments.